hadoop-common-project/hadoop-common/pom.xml

```diff
@@ -416,6 +416,12 @@
       <artifactId>lz4-java</artifactId>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>com.github.jnr</groupId>
```

Comment:
the license is tri EPL/GPL/LGPL license. Should be fine.

Reply:
Thanks for your review, and i have added com.github.jnr  to GPLv2 module in LICENSE-binary file.